lib/shared-method.js

```diff
@@ -236,24 +236,25 @@ function escapeRegex(d) {
  * @return {*}                 Coerced argument.
  */
 function coerceAccepts(uarg, desc) {
-  // If array, invoke for each member of the array.
-  if (Array.isArray(uarg)) {
-    return uarg.map(function(arg) {
-      return coerceAccepts(arg, desc);
-    });
-  }
-
   var name = desc.name || desc.arg;
   var targetType = desc.type;
-  var actualType = SharedMethod.getType(uarg);
-
-  // Target type may be an array. If so, expect the arg to be a member of that array,
-  // since we map this function over arrays.
   var targetIsArray = Array.isArray(targetType) && targetType.length === 1;
```

Comment:
this should really be named `targetIsTypedArray` or something that makes it clear that we are talking only about `['string']` and similar.